authors/apps/authentication/views.py

```diff
@@ -202,8 +202,8 @@ def post(self, request):
             return Response({"message": "The email you entered does not exist"}, status=status.HTTP_404_NOT_FOUND)
 
         # Get URL for client and include in the email for resetting password
-        client_url = request.META['HTTP_REFERER'].replace(
-            "login", "reset-password/")
+        client_url = request.META['HTTP_REFERER'] + \
+            "{}/".format("reset-password")
```

Comment:
@meshack-mbuvi I do not think we need to use .format() here as we are not adding a variable to the string.

Reply:
Wise saying. You have helped me learn something now.

Reply:
solved